### PR TITLE
redesign-custom-setting-page

### PR DESCRIPTION
**Pull request by sujitxtendify**
redesign figma link :- https://www.figma.com/file/kGSUK6bjNOtoEdsfbLpRNs/Simple-Calender?type=design&node-id=201-108&mode=design&t=nA2I1vsMnAc9omUz-0

you can check redesign :- https://www.screenpresso.com/=4KJlg

### CODE REVIEW

**Comment by rosinghal** [comment]
Is https://github.com/Xtendify/Simple-Calendar/pull/506 not needed?

**Comment by sujitxtendify** [comment]
No , because it had been merged and akhilesh sir told me to make new
branch.

**Comment by rosinghal** [comment]
In that case, please close that PR.


Cheers,
Rohit Singhal,
CEO of Xtendify <https://xtendify.com/>

Follow me:
Twitter <https://twitter.com/RoSinghal> | LinkedIn <https://www.linkedin.com/in/rosingha>

File: includes/admin/pages.php [hunk 1]
```diff
@@ -249,7 +249,7 @@ public function html()
 										<Span><?php _e('Please Rate Us !', 'google-calendar-events'); ?></Span>
 									</div>
 									<div class="simcal-mt-[5px] simcal-text-center simcal-font-normal simcal-text-base simcal-text-gray-500">
-										<Span><?php _e('If you like Simple Calendar please Rate Us', 'google-calendar-events'); ?> </Span>
+										<Span> <?php _e('If you like Simple Calendar please Rate Us', 'google-calendar-events'); ?> </Span>
```
Comment (Akhill2020) [inline includes/admin/pages.php:252]: @sujitxtendify Please make sure you dont push unnecessary changes.

File: includes/admin/post-types.php [hunk 2]
```diff
@@ -140,17 +140,15 @@ public function row_actions($actions, $post)
 	{
 		// Add a clear feed cache action link.
 		if ($post->post_type == 'calendar' && current_user_can('edit_posts')) {
-			$nonce_feed_actions = wp_create_nonce('nonce_feed_actions');
-
 			$actions['duplicate_feed'] =
 				'<a href="' .
-				esc_url(add_query_arg(['duplicate_feed' => $post->ID, 'nonce_feed_actions' => $nonce_feed_actions])) .
```
Comment (Akhill2020) [inline includes/admin/post-types.php:147]: @sujitxtendify Please take a pull and push changes. you replace changes done in a recent release.

File: includes/admin/post-types.php [hunk 3]
```diff
@@ -140,17 +140,15 @@ public function row_actions($actions, $post)
...
+				esc_url(add_query_arg(['duplicate_feed' => $post->ID])) .
 				'">' .
 				__('Clone', 'google-calendar-events') .
 				'</a>';
 			$actions['clear_cache'] =
 				'<a href="' .
-				esc_url(add_query_arg(['clear_cache' => $post->ID, 'nonce_feed_actions' => $nonce_feed_actions])) .
+				esc_url(add_query_arg(['clear_cache' => $post->ID])) .
```
Comment (Akhill2020) [inline includes/admin/post-types.php:151]: Here also revert the recent release changes done.

File: includes/admin/post-types.php [hunk 4]
```diff
@@ -167,7 +165,7 @@ public function row_actions($actions, $post)
 	public function bulk_actions()
 	{
 		// Check user has permission to edit
-		if ((!empty($_POST) && !wp_verify_nonce($_POST['_wpnonce'], '_wpnonce')) || !current_user_can('edit_posts')) {
+		if (!current_user_can('edit_posts')) {
```
Comment (Akhill2020) [inline includes/admin/post-types.php:168]: Here also revert the recent release changes done.

File: includes/admin/post-types.php [hunk 5]
```diff
@@ -235,7 +233,7 @@ public function edit_table_hooks()
 	 */
 	private function duplicate_feed($post_id)
 	{
-		if (!wp_verify_nonce($_REQUEST['nonce_feed_actions'], 'nonce_feed_actions') || !current_user_can('edit_posts')) {
```
Comment (Akhill2020) [inline includes/admin/post-types.php:238]: Here also revert the recent release changes done.
